index.html

```diff
@@ -223,9 +265,9 @@ <h3 class="skill-img ab"> <img src="img/skill55.png" alt=""> Language</h3>
         </div>
         <div class="card-skill">
             
-            <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
+            <h3 class="skill-img mab"> <img src="img/framwork56.png" alt=""> Framework</h3>
             <ul class="skill-group">
-                <li class="skillname spslill">
+                <li class="skillname">
                     <a href="#">Bootstrap</a>
```

Comment:
- [ ] Please as the last reviewer said   put the lists on the same line level, also the content of each list should be aligned to the left and not centered 
![image](https://user-images.githubusercontent.com/74744787/215293583-309c78a5-da99-49ab-a0e0-60b317e3e5aa.png)